### PR TITLE
Breadth first

### PR DESCRIPTION
**Pull request by herndont**
- [x] Write a breadth first traversal method which takes a Binary Tree as its unique input. 
- [x] Without utilizing any of the built-in methods available to your language, traverse the input tree using a Breadth-first approach; print every visited node’s value.
- [ ] Unit Tests:
          - Happy Path
          - Expected Failure
          - Edge Case

### CODE REVIEW

File: src/main/java/Tree/BinaryTree.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        queue.add(root);
+        while (queue.isEmpty()) {
+            Node poppedNode = queue.remove();
+            System.out.println(" " + poppedNode.data);
+            if (poppedNode.left != null) {
+                queue.add(poppedNode.left);
+            }
+            if (poppedNode.right != null) {
+                queue.add(poppedNode.right);
+            }
```
Comment (JessLovell) [inline src/main/java/Tree/BinaryTree.java:52]: What about when the node == null?
